pipeline-scripts/openshiftv4_on_osp.groovy

```diff
@@ -0,0 +1,233 @@
+			def openshift_machine_cidr = openshiftv4_properties['OPENSHIFT_MACHINE_CIDR']
+			def openshift_network_type = openshiftv4_properties['OPENSHIFT_NETWORK_TYPE']
+			def openshift_service_network = openshiftv4_properties['OPENSHIFT_SERVICE_NETWORK']
+			def openshift_host_prefix = openshiftv4_properties['OPENSHIFT_HOST_PREFIX']
+			def openshift_post_install_poll_attempts = openshiftv4_properties['OPENSHIFT_POST_INSTALL_POLL_ATTEMPTS']
+			def openshift_toggle_infra_node = openshiftv4_properties['OPENSHIFT_TOGGLE_INFRA_NODE']
+			def openshift_toggle_workload_node = openshiftv4_properties['OPENSHIFT_TOGGLE_WORKLOAD_NODE']
+			def machineset_metadata_label_prefix = openshiftv4_properties['MACHINESET_METADATA_LABEL_PREFIX']
+			def openshift_infra_node_instance_type = openshiftv4_properties['OPENSHIFT_INFRA_NODE_INSTANCE_TYPE']
+			def openshift_workload_node_instance_type = openshiftv4_properties['OPENSHIFT_WORKLOAD_NODE_INSTANCE_TYPE']
+			def openshift_infra_node_volume_size = openshiftv4_properties['OPENSHIFT_INFRA_NODE_VOLUME_SIZE']
+			def openshift_infra_node_volume_type = openshiftv4_properties['OPENSHIFT_INFRA_NODE_VOLUME_TYPE']
+			def openshift_infra_node_volume_iops = openshiftv4_properties['OPENSHIFT_INFRA_NODE_VOLUME_IOPS']
+			def openshift_workload_node_volume_size = openshiftv4_properties['OPENSHIFT_WORKLOAD_NODE_VOLUME_SIZE']
+			def openshift_workload_node_volume_type = openshiftv4_properties['OPENSHIFT_WORKLOAD_NODE_VOLUME_TYPE']
+			def openshift_workload_node_volume_iops = openshiftv4_properties['OPENSHIFT_WORKLOAD_NODE_VOLUME_IOPS']
+			def openshift_prometheus_retention_period = openshiftv4_properties['OPENSHIFT_PROMETHEUS_RETENTION_PERIOD']
+			def openshift_prometheus_storage_class = openshiftv4_properties['OPENSHIFT_PROMETHEUS_STORAGE_CLASS']
+			def openshift_prometheus_storage_size = openshiftv4_properties['OPENSHIFT_PROMETHEUS_STORAGE_SIZE']
+			def openshift_alertmanager_storage_class = openshiftv4_properties['OPENSHIFT_ALERTMANAGER_STORAGE_CLASS']
+			def openshift_alertmanager_storage_size = openshiftv4_properties['OPENSHIFT_ALERTMANAGER_STORAGE_SIZE']
+			def kubeconfig_auth_dir_path = openshiftv4_properties['KUBECONFIG_AUTH_DIR_PATH']
+			
+			// Install cluster using the payload captured at the build trigger url when scale_ci_build_trigget is set
+			if ( scale_ci_build_trigger.toBoolean() ) {
+				sh "curl ${scale_ci_build_trigger_url}/payload -o /tmp/payload"
+				sh "curl ${scale_ci_build_trigger_url}/build.status -o /tmp/status"
+				status = readFile "/tmp/status"
+				if ( status.toString().trim().equals("PROCEED") ) {
+					println "Build status is set to ${status}, proceeding with the cluster build"
+					openshift_install_release_image_override = readFile "/tmp/payload"
+				} else {
+					println "Build status is set to ${status}, exiting without building the cluster"
+					return 0
+				}
+			}
+
+			try {
+				openshiftv4_build = build job: 'ATS-SCALE-CI-TEST',
```

Comment:
Just a reminder, we need to switch the name of the job back to the name specified in JJB template.